mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/query/parsing/nativequery/NativeQueryParserTest.java

```diff
@@ -279,4 +280,21 @@ public void shouldFindeCountQueryWithLogicalOperatorNOT() {
 		assertThat( queryDescriptor.getProjection() ).isNull();
 		assertThat( queryDescriptor.getOrderBy() ).isNull();
 	}
+
+	@Test
+	@TestForIssue(jiraKey = "OGM-900")
+	public void shouldSupportDotInCollectionName() {
```

Comment:
yes I added one, hoping it's in the best location
cf https://github.com/francoisledroff/hibernate-ogm/blob/93c0bc6733cbf07dfa4c5593598b1907cbb98474/mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/datastore/CollectionNamingValidationTest.java#L58